app/Models/Preference.php

```diff
@@ -11,16 +11,30 @@
 /**
  * @property int $id
  * @property int $user_id
- * @property array $preferences
+ * @property array|null $preferences
+ * @property string|null $city
+ * @property float|null $latitude
+ * @property float|null $longitude
+ * @property int $radius_km
  * @property-read User $user
  */
 class Preference extends Model
 {
     use HasFactory;
 
-    protected $fillable = ["user_id", "preferences"];
+    protected $fillable = [
+        "user_id",
+        "preferences",
+        "city",
+        "latitude",
+        "longitude",
+        "radius_km",
```

Comment:
```suggestion
        "radius_in_km",
```
That should be more readable